packages/gitbook/src/app/(space)/(content)/[[...pathname]]/page.tsx

```diff
@@ -89,6 +89,7 @@ export default async function Page(props: {
                         // Display the page feedback in the page footer if the aside is not visible
                         withPageFeedback && !page.layout.outline
                     }
+                    searchParams={searchParams}
```

Comment:
Related to the approach I'm suggesting with a react context, we could avoid passing these search search params everywhere, and instead wrap it all in a client component that uses `useSearchParams` from next